src/integrationTest/java/org/opensearch/test/framework/cluster/TestRestClient.java

```diff
@@ -122,6 +123,12 @@ public HttpResponse getAuthInfo( Header... headers) {
 		return executeRequest(new HttpGet(getHttpServerUri() + "/_opendistro/_security/authinfo?pretty"), headers);
 	}
 
+	public void assertCorrectCredentials() {
```

Comment:
In its usage its a test to check that the credentials work, but its really calling the getAuthInfo api and making sure its got a 200 response.  It would be more useful if we could provide an expected username or other value as a parameter to ensure its running as the `correct` context, what do you think?

Reply:
done